_meta/kibana/default/dashboard/apm-dashboards.json

```diff
@@ -21,7 +21,7 @@
       "type": "visualization",
       "version": 2,
       "attributes": {
-        "title": "APM: Top Apps by Response Time",
+        "title": "APM: Top Applications by Response Time [APM]",
```

Comment:
Could remove the `APM:` in the beginning, since you added it at the end and to be consistent?